pkg/pipelines/environment_test.go

```diff
@@ -237,6 +237,20 @@ func assertFileExists(t *testing.T, testFs afero.Fs, path string) {
 	}
 }
 
+func assertDirExists(t *testing.T, testFs afero.Fs, path string) {
+	t.Helper()
+	exists, err := afero.Exists(testFs, path)
```

Comment:
You don't need to test for `Exists` and `DirExists` as they both fail if the path doesn't exist.